GUI/Main.Designer.cs

```diff
@@ -64,7 +65,7 @@ private void InitializeComponent()
             this.splitContainer1 = new System.Windows.Forms.SplitContainer();
             this.ModList = new CKAN.MainModListGUI();
             this.Installed = new System.Windows.Forms.DataGridViewCheckBoxColumn();
-            this.UpdateCol = new System.Windows.Forms.DataGridViewCheckBoxColumn();
+            this.Update = new System.Windows.Forms.DataGridViewCheckBoxColumn();
```

Comment:
Please revert, or expand name to UpdateColumn or UpdateSelection or some such. "Update" gives us build warnings.

    Main.Designer.cs(1614,44): warning CS0108: `CKAN.Main.Update' hides inherited member `System.Windows.Forms.Control.Update()'. Use the new keyword if hiding was intended
    /usr/lib/mono/4.5-api/System.Windows.Forms.dll (Location of the symbol related to previous warning)
    Main.Designer.cs(1600,44): warning CS0169: The private field `CKAN.Main.UpdateCol' is never used